core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/UpgradeOption.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response;
+
+public class UpgradeOption {
```

Comment:
On API related classes we are using v4 postfix, please rename this class accordingly